screen-wake-lock/wakelock-onrelease.https.html

```diff
@@ -20,4 +20,20 @@
     assert_false(ev.cancelable);
   });
 }, "Test onreleased event's basic properties");
+
+promise_test(async t => {
+  await test_driver.set_permission({ name: 'screen-wake-lock' }, 'granted', false);
```

Comment:
Oh sorry, one more nit... what doest the `false` do at the end there? maybe add:

```JS
await test_driver.set_permission({ name: 'screen-wake-lock' }, 'granted',  /*whatever=*/ false);
```

So it's clear

Reply:
It's the `one_realm` parameter documented here: https://web-platform-tests.org/writing-tests/testdriver.html#set-permission

I see that some tests just don't set it at all, I'll see if everything works and update all screen wake lock tests separately in this case.